src/parser/RdfParser.h

```diff
@@ -265,7 +268,7 @@ class TurtleParser : public RdfParserBase {
   virtual bool statement();
 
   // Log error message (with parse position) and throw parse exception.
-  [[noreturn]] void raise(std::string_view error_message) {
+  [[noreturn]] void raise(std::string_view error_message) const {
```

Comment:
While cleaning up, 
Can we also move this function into the `.cpp` file?

Reply:
(maybe do this for all function > 4 lines or so in this file as a cleanup measure).